src/features/components/molecules/image-text-boxes/types.ts

```diff
@@ -0,0 +1,22 @@
+import { ReactElement } from 'react'
+import { TFlexDirection } from '../../../types'
+import { TString } from 'types/generics'
+
+export type TAcuityFeatures = {
+    mobileFlexDirection: TFlexDirection
+    data: TContentData[]
+}
+interface TContentData {
+    id: number
+    direction: TFlexDirection
+    image: ReactElement
+    title?: TString
+    subtitle?: TString
+    info_title?: TString
+    system_url?: string | ((config: Partial<SystemUrlItemConfig>) => string)
```

Comment:
is `Partial` required here since `SystemUrlItemConfig` is having `is_eu`?
```suggestion
    system_url?: string | ((config: SystemUrlItemConfig) => string)
```